### DISCUSSION TITLE
Smartwatch for Standalone Data Monitor for Dexcom G6

### DISCUSSION THREAD

**Discussion by enginemr2** [NightscoutFoundation/xDrip#2192] [General]
For the last 5 years I have been using the Sony SWR50 Smartwatch for my standalone data monitor from the G5 and G6 transmitters. It has worked perfectly for getting the data every 5 minutes. For my monitoring the 5-minute intervals are very important for getting the data gradient during my aerobic workouts. Are there any newer smartwatches that get the data for every 5-minute interval?

**Comment by Navid200** [comment]
Every 5 minutes, G5 and G6 only (not for Libre):
https://bluejay.website/shop/product/bluejay-gts-26

**Reply by enginemr2** [reply]
I have checked the Bluejay. It appears to have many good features, but I still prefer the SWR50 transflective display. Also the bluetooth range to the smartphone for the SWR50 seems to be better than most. I have been able to relay data to my smartphone anywhere that is line-of-site.
